File: docs/reference/snippets/python/a2v/video_generate.py
import requests
import json

url = "https://openapi.surreal-ai.com/v1/a2v/video.generate"

payload = json.dumps({
  "pose_id": "liza_a0008",
  "audio_url": "https://example.com/audio.wav"
})
headers = {
  'Authorization': 'Bearer bee03bed6352400383d9a51ba325af2f',
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

File: docs/reference/snippets/java/a2v/video_generate.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("application/json");
RequestBody body = RequestBody.create(mediaType, "{\n    \"pose_id\": \"liza_a0008\",\n    \"audio_url\": \"https://example.com/audio.wav\"\n}");
Request request = new Request.Builder()
  .url("https://openapi.surreal-ai.com/v1/a2v/video.generate")
  .method("POST", body)
  .addHeader("Authorization", "Bearer bee03bed6352400383d9a51ba325af2f")
  .addHeader("Content-Type", "application/json")
  .build();
Response response = client.newCall(request).execute();
